app/controllers/application_controller.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  before_action do
-    # Unless the request format is explicitly json Rails will mitigate the responsability to CSR to handle it.
-    render 'components/index' unless valid_api_request?
-  end
+  # before_action do
```

Comment:
After development it would be better to uncomment this and make the required e2e tests in a complete environment.